assets/locales/en/translation.json

```diff
@@ -170,6 +170,7 @@
   "Selector": "Selector",
   "Interactions are not available for this element": "Interactions are not available for this element",
   "usingXPathNotRecommended": "Using XPath locators is not recommended and can lead to fragile tests. Ask your development team to provide unique accessibility locators instead!",
+  "usingSwitchContextRecommended": "Multiple contexts are detected. It is advised to switch context to WEBVIEW if you easily want to select elements like you would do on the web.",
```

Comment:
Only 'en' resource is fine 👍 
https://github.com/appium/appium-desktop/blob/52eb9c1c8076fe059ead31df24c62b4556e3ea73/CONTRIBUTING.md#localization

Reply:
ahh, so I can remove all the other languages?

Reply:
exactly